### DISCUSSION TITLE
Top-down tutorial

### DISCUSSION THREAD

**Discussion by nnalpas** [smith-chem-wisc/MetaMorpheus#2189] [Q&A]
Hello,
I am trying to test MetaMorpheus for a top-down measurement.
Is there a tutorial (and maybe test dataset) that I could use to check the correct parameters to use in MetaMorpheus?
Best regards,

**Comment by trishorts** [comment]
If you select top-down as the Protease, most default parameters will be adjusted automatically to the appropriate numbers.
